### PR TITLE
Re-refactor Mokulua

### PR DESCRIPTION
**Pull request by kylemccreery**
## Description

The refactor of the Mokulua firmware removed the support for one of the 2 possible PCBs that this split board can use. This readds support for the second PCB while keeping a couple of the changes that were suggested by that refactor.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Readding removed support for PCB

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]
Conflicts need to be resolved where the branch was reused from a previous PR.

**Comment by kylemccreery** [comment]
@zvecr Fixed. Thanks. Sorry I missed that.

File: keyboards/mechwild/mokulua/mirrored/mirrored.c
Comment (drashna) [inline keyboards/mechwild/mokulua/mirrored/mirrored.c:31]: Please revert this change.  This should be the kb level function, NOT the user level. 
```suggestion
	oled_rotation_t oled_init_kb(oled_rotation_t rotation) {
```

File: keyboards/mechwild/mokulua/standard/standard.c
Comment (drashna) [inline keyboards/mechwild/mokulua/standard/standard.c:31]:
```suggestion
	oled_rotation_t oled_init_kb(oled_rotation_t rotation) {
```

**Review by drashna — CHANGES_REQUESTED** [review]
It looks like you have changes to the submodules here.  These need to be reverted before the PR can be approved. 

Rebasing sometimes will fix this. Otherwise, you'd want to do something like this to fix them:
```sh
git checkout master
git submodule foreach 'git log -n1 --oneline'
# remember the first abcdef1234 on the submodules that have an issue
git checkout @@your-branch@@
# for each of the problematic repo's:
cd lib/@@name@@
git checkout abcdef1234
```

File: keyboards/mechwild/mokulua/mirrored/config.h
Comment (drashna) [inline keyboards/mechwild/mokulua/mirrored/config.h:60]:
```suggestion
```

**Review by drashna — APPROVED** [review]

File: keyboards/mechwild/mokulua/standard/standard.c
Comment (fauxpark) [inline keyboards/mechwild/mokulua/standard/standard.c:31]: Mixed tabs and spaces here, and in other files.

File: keyboards/mechwild/mokulua/standard/keymaps/default/keymap.c
Comment (fauxpark) [inline keyboards/mechwild/mokulua/standard/keymaps/default/keymap.c:14]:
```suggestion
    _FN2,
    _FN3
```

File: keyboards/mechwild/mokulua/mirrored/keymaps/via/keymap.c
Comment (fauxpark) [inline keyboards/mechwild/mokulua/mirrored/keymaps/via/keymap.c:14]:
```suggestion
    _FN2,
    _FN3
```

File: keyboards/mechwild/mokulua/mirrored/keymaps/default/keymap.c
Comment (fauxpark) [inline keyboards/mechwild/mokulua/mirrored/keymaps/default/keymap.c:14]:
```suggestion
    _FN2,
    _FN3
```

File: keyboards/mechwild/mokulua/standard/keymaps/via/keymap.c
Comment (fauxpark) [inline keyboards/mechwild/mokulua/standard/keymaps/via/keymap.c:14]:
```suggestion
    _FN2,
    _FN3
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by kylemccreery** [comment]
@zvecr @fauxpark Anything else that needs to be changed on this for a rereview?

**Review by noroadsleft — APPROVED** [review]
Slight `__attribute__ ((weak))` :heavy_check_mark:.

I diff'd the previous `mirrored`/`standard` implementation and came to the same conclusion @waffle87 seems to have done, but if the keyboard maintainer says some of the supported hardware doesn't work on the post-refactor version, I'm inclined to believe them.

**Review by noroadsleft — APPROVED** [review]
Took another look at this and spotted an error in the original refactor that this PR solves.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**
